### ISSUE TITLE
Cannot load the pre-trained model

### ISSUE THREAD

**Issue by findalexli** [OxWearables/ssl-wearables#8]
I used the provided torch hub way to load the harnet5/10/30 model, but I wanted to be able to make inference in Rowland labels, so I download and attempted to load the Rowland pre-trained model. However, it seemed like the Rowland Pre-trained model has a different model architecture than the Torch hub models. Can you please clarify? 

Another adjacent question is, does 'Rowland Pre-trained' model mean a model that 
1) Trained from scratch using the Rowland dataset
2) Trained on the UK-biobank contrastively and fine-tuned on the Rowland dataset?

**Comment by angerhang** [comment]
Hey @findalexli thanks for getting in touch. 

For your questions: 

* Rowland pretrained should have the same architecture as harnet10?
* `Rowland pre-trained` means that the model is pre-trained on the Rowland dataset. As the focus of this project was to release large-scale pre-trained models instead of releasing models for activity recognition. 

I think you would like to have a harnet5/10/30 and fine-tuned on the Rowlands to make inference? If that's the case, we will have to make that available as all the fine-tuned models are made public yet. I think we have the fine-tuned models for harnet10 but not sure for other models with different time windows.

**Comment by angerhang** [comment]
If you have some labelled data from your side, you can easily fine-tune the pre-trained harnets.

If not, depending on the requirements for your requests, it might take some time for us to make those models available to you :D Ranging from 1 day - $\infty $

**Comment by aidendoherty** [comment]
Dear @findalexli - many thanks for your interest. Unfortunately we do not have permissions to release the Rowlands dataset (unlike CAPTURE-24 for example). It's currently not a priority for us to release the associated downstream model for this at present. However, if we receive a substantial number of similar requests we are of course happy to revise this in future.

**Comment by findalexli** [comment]
Thank you so much for the prompt reply! This clarifies things up. 

It sounds like for the time being, to do so I will need to contact Alex Rowlands (or anyone else, let me know if you have any suggestions) to get access to the dataset, such that I can make inference on a Biobank-pretrained and Rowlands-finetuned model whose accuracy is mentioned in the paper.
